test/unit/modules/loader.skeleton.js

```diff
@@ -270,47 +270,4 @@ describe('loader', function () {
 
 		it('should call callback with result = undefined');
 	});
-
-	describe('internal', function () {
-
-		describe('statusPing', function () {
-
-			it('should call modules.blocks.lastBlock.isFresh');
-
-			it('should return modules.blocks.lastBlock.isFresh result');
-		});
-	});
-
-	describe('shared', function () {
-
-		describe('status', function () {
-
-			it('should call callback with error = null');
-
-			it('should call callback with result containing loaded = __private.loaded');
-
-			it('should call callback with result containing now = __private.lastBlock.height');
-
-			it('should call callback with result containing blocksCount = __private.total');
-		});
-
-		describe('sync', function () {
-
-			it('should call callback with error = null');
-
-			it('should call callback with result containing blocks = __private.blocksToSync');
-
-			it('should call modules.blocks.lastBlock.get');
-
-			it('should call callback with result containing height = modules.blocks.lastBlock.get result');
-
-			it('should call modules.system.getBroadhash');
-
-			it('should call callback with result containing broadhash = modules.system.getBroadhash result');
-
-			it('should call modules.peers.getConsensus');
-
-			it('should call callback with result containing consensus = modules.peers.getConsensus result');
-		});
-	});
```

Comment:
The whole file `test/unit/modules/loader.skeleton.js` just contains test statements but not test cases. 

```
describe('loader', function () {

	describe('constructor', function () {

		describe('library', function () {

			it('should assign logger');

			it('should assign db');

			it('should assign network');

			it('should assign schema');

			it('should assign sequence');

			it('should assign bus');

			it('should assign genesisblock');

			it('should assign balancesSequence');
```

If we are not going to implement these, then we should remove this file completely.

Reply:
The module is not removed. I remove just the deprecated endpoint `loader`. The module and the logic still being used in the code.